assets/tests/components/ladderPage.test.tsx

```diff
@@ -287,10 +287,10 @@ describe("LadderPage", () => {
       </StateDispatchProvider>
     )
 
-    expect(wrapper.find(".m-ladder-page__tab-save-button").length).toBe(0)
+    expect(result.queryByTestId("save-button")).toBeNull()
```

Comment:
I think this needs to be updated to reference the title instead (the result should be null either way but right now I don't think it's actually an effective test if the test ID is no longer in place).